galaxy/importer/tests/test_linters.py

```diff
@@ -104,3 +106,87 @@ def test_yamllint_fail():
         ]
         expected = [s.format(fp.name) for s in expected]
         assert sorted(result) == sorted(expected)
+
+
+ANSIBLELINT_TEST_FILE_OK = """---
+- name: Add mongodb repo apt_key
+  become: true
+  apt_key: keyserver=hkp
+"""
+
+
+def test_ansiblelint_ok():
+    try:
+        temp_root = tempfile.mkdtemp()
```

Comment:
Use pytest fixtures for this, it doesn't belong to the test:
```python
import pytest
...
@pytest.fixture
def temp_root():
    try:
        tmp = tempfile.mkdtemp()
        yield tmp
    finally:
        shutil.rmtree(tmp)
...
def test_ansiblelint_ok(temp_root):
    ...
    assert stuff without any try/except/finally here
    ...
```

Reply:
Updated